source/core/index.ts

```diff
@@ -199,6 +200,7 @@ export default class Request extends Duplex implements RequestEvents<Request> {
 		this._unproxyEvents = noop;
 		this._triggerRead = false;
 		this._cancelTimeouts = noop;
+		this._removeListeners = noop;
```

Comment:
@szmarczak Unrelated, but why are we assigning here instead of just giving the class fields a new value directly?

Reply:
I guess I'm just not used to this. I agree, it looks much better if the values were given directly :)